connectors/src/api/webhooks/webhook_slack_reaction.ts

```diff
@@ -0,0 +1,106 @@
+import { Request, Response } from "express";
+
+import { botAnswerMessageWithErrorHandling } from "@connectors/connectors/slack/bot";
+import { APIErrorWithStatusCode } from "@connectors/lib/error";
+import logger from "@connectors/logger/logger";
+import { withLogging } from "@connectors/logger/withlogging";
+
+type SlackInteractionPayload = {
+  team?: {
+    id: string;
+    domain: string;
+  };
+  channel?: {
+    id: string;
+    name: string;
+  };
+  message?: {
+    ts: string;
+    thread_ts?: string;
+  };
+  user?: {
+    id: string;
+  };
+  state?: {
+    values: {
+      [key: string]: {
+        [key: string]: {
+          selected_option?: {
+            value: string;
+          };
+        };
+      };
+    };
+  };
+};
+
+type SlackWebhookResBody =
+  | { challenge: string }
+  | null
+  | APIErrorWithStatusCode;
+
+const _webhookSlackReactionsAPIHandler = async (
```

Comment:
Why is it called Reaction? Is it because it's called like that from Slack API? Could we rename to `Interaction` instead?
Because on the app reaction is for Message Reaction <> Emojis, and I'm afraid this could be confusing.